src/main/java/bio/terra/cli/command/resource/MoveToFolder.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.cli.command.resource;
+
+import static bio.terra.cli.command.resource.ListTree.TERRA_FOLDER_ID_PROPERTY_KEY;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.command.shared.WsmBaseCommand;
+import bio.terra.cli.command.shared.options.Format;
+import bio.terra.cli.command.shared.options.ResourceName;
+import bio.terra.cli.command.shared.options.WorkspaceOverride;
+import bio.terra.cli.serialization.userfacing.UFResource;
+import java.util.Map;
+import java.util.UUID;
+import picocli.CommandLine;
+
+/** This class corresponds to the third-level "terra resource move" command. */
+@CommandLine.Command(name = "move", description = "move resource to a folder.")
+public class MoveToFolder extends WsmBaseCommand {
+  @CommandLine.Mixin ResourceName resourceNameOption;
+
+  @CommandLine.Option(names = "--folder-id", required = true, description = "folder id")
+  public UUID folderId;
```

Comment:
Can we let users provide folder name instead of folder Id? I also don't think we should expose folder UUID in `terra folder tree` either

Reply:
Actually I realize we might need uuid for the data collection management use case. But it would still be nicer to pass folder name when moving a resource

Reply:
it does expose uuid in terra folder tree

folder name is not unique in a workspace so it's a lot more work

Reply:
Good point thats a bummer. Can you update the command description to something like "folder UUID" instead of "folder id"